packages/strapi-admin/controllers/role.js

```diff
@@ -13,4 +13,10 @@ module.exports = {
       data: role,
     };
   },
+  async find(ctx) {
+    const roles = await strapi.admin.services.role.find({ _limit: -1 });
```

Comment:
We could create a findAll service so the controller staus as dumb as possible

Reply:
Yes I agree